backend/src/repositories/PricesRepository.ts

```diff
@@ -65,7 +65,7 @@ class PricesRepository {
     
     try {
       await DB.query(`
-        INSERT INTO prices(time,             USD, EUR, GBP, CAD, CHF, AUD, JPY)
+        INSERT IGNORE INTO prices(time,             USD, EUR, GBP, CAD, CHF, AUD, JPY)
```

Comment:
Better catch errors you want to silence rather than silence everything, see for example: https://github.com/mempool/mempool/blob/master/backend/src/repositories/BlocksRepository.ts#L180

But if your main issue is duplicated entries, here it would make more sense to use `ON UPDATE` to update the latest price entry in the db.